zigate/cluster/on_off.py

```diff
@@ -0,0 +1,22 @@
+import struct
+
+from .core import Attribute, Cluster
+
+
+def unpack_string(data):
+    return data.decode('utf-8')
+
+def struct_unpack(fmt):
+    st = struct.Struct(fmt)
+    def _(data):
+        return st.unpack(data)[0]
+    return _
+
+
+class OnOff(Cluster):
```

Comment:
I wouldn't mind renaming this to `Swtich`,  a slight departure from the Zigate/Zigbee API, but more natural.

Reply:
In that case, what do you want to name cluster 0x0007: On/Off Switch Configuration?
On/Off is less ambiguous than Switch, but if you insist, I am fine with either :)

Reply:
mmm ok, guess you're right. Let's leave it at 'OnOff' then.